public/testem/testem_client.js

```diff
@@ -266,6 +267,9 @@ var Testem = {
         case 'stop-run':
           self.emit('after-tests-complete');
           break;
+        default:
+          self.emit(type, message.data);
```

Comment:
emitting all events from window.message as now part of testem's public API seems error prone. Remember, lots of applications actually use these for communication.

Thoughts on only propagating events which we expect rather then all?

One approach, would be to namespace these new events, for example, one could prefix them with `socket-io:${type}` or something, then once we hit this default switch, we only emits those which `startWith('socket-io:')`